.github/workflows/alpha-release.yml

```diff
@@ -16,77 +16,82 @@ jobs:
       changelog: ${{ steps.changelog.outputs.changelog }}
       release_id: ${{ steps.release.outputs.id }}
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         with:
           fetch-depth: 0
           token: ${{ secrets.PRIVATE_TOKEN }}
 
-      - uses: actions/setup-node@v3
+      - name: Set up version.json
+        run: echo "{"version":$(git describe --tags --abbrev=0)}" > version.json
+
+      - uses: actions/setup-node@v4
         with:
           node-version: 16
       - run: npm install conventional-changelog-conventionalcommits
 
       - name: Create changelog
         id: changelog
-        uses: TriPSs/conventional-changelog-action@v3.17.0
+        uses: TriPSs/conventional-changelog-action@v5.2.1
         with:
           github-token: ${{ secrets.PRIVATE_TOKEN }}
           git-user-name: 'WynntilsBot'
           git-user-email: 'admin@wynntils.com'
           pre-commit: ./.github/.pre-commit.js
           config-file-path: ./.github/.config.js
+          version-file: ./version.json
           skip-version-file: true
           skip-git-pull: true
           pre-release: true
           pre-release-identifier: alpha
+          release-count: 5
```

Comment:
What does this mean?

Reply:
It's copied from release, it's basically how many releases it keeps in CHANGELOG.md

Reply:
Ah.